packages/maitred/internal/auth/auth.go

```
@@ -0,0 +1,45 @@
+	if err != nil {
+		return nil, err
+	}
+	defer func(Body io.ReadCloser) {
+		err = Body.Close()
+		if err != nil {
+			slog.Error("Error closing body", "err", err)
+		}
+	}(resp.Body)
+	if resp.StatusCode != 200 {
+		body, _ := io.ReadAll(resp.Body)
+		return nil, fmt.Errorf("failed to auth: " + string(body))
```

golangci-lint (1.64.8)

printf: non-constant format string in call to fmt.Errorf

(govet)